content/en/about/feature-stages/index.md

```diff
@@ -82,13 +82,10 @@ Below is our list of existing features and their current phases. This informatio
 | [Standalone Operator](/docs/setup/install/operator/) | Beta
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/) | Stable
 | [Kubernetes: Istio Control Plane Installation](/docs/setup/) | Stable
-| [Attribute Expression Language](https://istio.io/v1.6/docs/reference/config/policy-and-telemetry/expression-language/) | Stable
 | Mixer Out-of-Process Adapter Authoring Model | deprecated
-| [Multicluster Mesh](/docs/setup/install/multicluster/) | Alpha
-| [Kubernetes: Istio Control Plane Upgrade](/docs/setup/) | Beta
-| Consul Integration | Alpha
+| [Multicluster Mesh](/docs/setup/install/multicluster/) | Beta
```

Comment:
@nmittler did this make beta? I thought we were proceeding - unfortunately - with alpha for this feature.

Reply:
Yes it did to my best knowledge